README.md

```diff
@@ -44,7 +44,7 @@ Please note that we currently don't provide native binaries for macOS, because e
 
 ### Other
 
-On other operating systems you can use this tool using Java. For the you need Java >= 17 installed.
+On other operating systems you can use this tool using Java. For that you need Java >= 17 installed.
```

Comment:
On other operating systems, you can use this tool with Java. For that, you need Java version 17 or higher installed.